### ISSUE TITLE
Empty training data

### ISSUE THREAD

**Issue by GeorgeLiang3** [mitmath/18337#16]
Hi Chris, @ChrisRackauckas 

I am trying to follow your tutorial on youtube, it's really nice and helpful.  I got a bit lost here at `data = Iterators.repeated((), 5000)`, for example, the Hooks law, I am quite new to NN, I was expecting the training data to be (position, force) pairs. Could you explain a bit ? Thanks

**Comment by ChrisRackauckas** [comment]
Yes. This is fairly odd but simple. The Flux training loop wants data. We don't need to "iterate over data" though, since we enclosed all of our training data into the loss function. So a quick hack is to make an empty data iterator that does 5000 iterations, which is a way to make the training loop run for 5000 iterations. It's somewhat of a small hack.

**Comment by GeorgeLiang3** [comment]
Thanks very much for the explaination.
